src/components/shared/lazy_loaded_image.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class LazyLoadedImage extends React.PureComponent {
+  imageRef = React.createRef();
+
+  componentDidMount() {
+    const { current } = this.imageRef;
+    const { src } = this.props;
+
+    if (current && src) {
+      current.src = src;
+    }
+  }
+
+  render() {
+    return <img {...this.props} src={null} ref={this.imageRef} />;
+  }
+}
```

Comment:
Not sure what this component is doing...

Reply:
> Not sure what this component is doing...

This delays image loading to after the component has already loaded, to make the first page interaction quicker